components/brave_ads/common/search_result_ad_util.cc

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/common/search_result_ad_util.h"
+
+#include "base/strings/string_piece.h"
+#include "url/gurl.h"
+#include "url/third_party/mozilla/url_parse.h"
+
+namespace brave_ads {
+
+namespace {
+
+constexpr char kSearchAdsConfirmationHost[] =
+    "search-ads-confirmation.brave.com";
```

Comment:
should be added to the cert pinning list; see https://github.com/brave/internal/wiki/Pull-request-security-audit-checklist#hostnames

Reply:
added